temporian/core/operators/test/BUILD

```diff
@@ -379,3 +379,14 @@ py_test(
         "//temporian/test:utils",
     ],
 )
+
+py_test(
+    name = "test_drop",
+    srcs = ["test_drop.py"],
+    srcs_version = "PY3",
+    deps = [
+        "//temporian/implementation/numpy/data:io",
+        "//temporian/test:utils",
+        "//temporian/core/operators:drop",

```

Comment:
unneeded (you're calling drop from the EventSet, not importing it from its module)